content/advanced-networking/secondary_cidr/configure-cni.md

```diff
@@ -13,7 +13,11 @@ kubectl describe daemonset aws-node --namespace kube-system | grep Image | cut -
 ```
 Here is a sample response
 ```
+<<<<<<< HEAD
+amazon-k8s-cni:1.2.1
+=======
 amazon-k8s-cni:1.4.1
+>>>>>>> upstream/master
```

Comment:
what is the need for this to be changed?

Reply:
That was a conflict. Resolved